### DISCUSSION TITLE
Forge test -m deprecated

### DISCUSSION THREAD

**Discussion by ChefAharoni** [Cyfrin/foundry-full-course-cu#171] [Ideas]
Hey all,

Looks like `forge test -m` as used in lesson 7 for testing a particular function, no longer works.

Looking through the [docs](https://book.getfoundry.sh/faq#how-do-i-run-specific-tests), it seems that `forge test --mt` (or `forge test --match-test`) replaces the `-m`.

I think this should be added to the readme & on the youtube video at the correct timestamp to help confused watchers (me amongst them) who don't understand why doesn't it work.

Edit:
Seems as `forge snapshot -m` doesn't work as well, but the fix is the same: `forge snapshot --mt`.

**Comment by RichardPalladino** [comment]
I confirm that `forge test -m` did not work, so I have been running  with `forge test --match-test` as well.

**Comment by thurendous** [comment]
Thanks for this information. I was wondering about this too.

**Comment by dalteko** [comment]
I've been using forge test --mt 

That's been working for me when I ran into the same issue!

**Comment by Teoaks** [comment]
Found the same issue. Thanks for the solution!

**Comment by DukeLurkin** [comment]
Thank you for this!

**Comment by cromewar** [comment]
Thanks for posting this @ChefAharoni

**Comment by GauravB2004** [comment]
Thanks for the solution, I was getting the error unexpected argument -m found.

**Comment by albin-george-kurian** [comment]
i would like to conform this information is valid,  i were also getting this same kind of error
i replaced ``` -m``` with ``` -mt ```
